specification/ai/Azure.AI.Projects/agents/tools/models.tsp

```diff
@@ -52,8 +52,8 @@ model BingGroundingToolDefinition extends ToolDefinition {
   @doc("The object type, which is always 'bing_grounding'.")
   type: "bing_grounding";
 
-  @doc("The list of connections used by the bing grounding tool.")
-  bing_grounding: ToolConnectionList;
+  @doc("The list of configurations to be used by the bing grounding tool.")
+  bing_grounding: BingGroundingSearchConfiguration[];
```

Comment:
I think this should be 

```tsp
bing_grounding: {
   @removed(Versions.V2025_05_01)
   connections: ToolConnection[]
   @added(Versions.V2025_05_01)
   search_configurations: BingGroundingSearchConfiguration[],
}
```

The idea is that the TypeSpec description represents the current state of the service, including the versions. While the service still supports versions with connections, the typespec should describe them both.

Versioning docs for reference: https://typespec.io/docs/libraries/versioning/guide/